servicedirectory/quickstart_test.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os import environ
+import pytest
+import quickstart
+from google.api_core import exceptions
+from google.cloud import servicedirectory_v1beta1
+
+PROJECT_ID = environ['GOOGLE_CLOUD_PROJECT']
+LOCATION_ID = environ['GOOGLE_CLOUD_LOCATION']
```

Comment:
```suggestion
LOCATION_ID = 'us-east1'
```

This seems to be what other tests do for location

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/ee5be6df3f8e43bd7798bc56a39b7c5f273ca90f/vision/cloud-client/product_search/product_search_test.py#L22-L23